lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -55,10 +56,35 @@ def check_question_author(user, content):
         except KeyError:
             return False
 
+    def check_team_member(user, content):
+        """
+        If the content has a commentable_id, verifies that either it is not associated with a team,
+        or if it is, that the user is a member of that team.
+        """
+        if not content:
+            return False
+        try:
+            commentable_id = content['commentable_id']
+            request_cache_dict = RequestCache.get_request_cache().data
+            cache_key = "django_comment_client.check_team_member.{}.{}".format(user.id, commentable_id)
+            if cache_key in request_cache_dict:
+                return request_cache_dict[cache_key]
+            team = CourseTeam.objects.get(discussion_id=commentable_id)
+            passes_condition = team.users.filter(id=user.id).count() > 0
+            request_cache_dict[cache_key] = passes_condition
+        except KeyError:
+            passes_condition = False
```

Comment:
I think only in tests, when the commentable_id isn't mocked out (I hit that a lot when getting all the tests to pass). It's the same pattern as all the other "check" methods.